src/WebJobs.Script/Binding/ServiceBus/ServiceBusValidator.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Microsoft.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public interface INamespaceManager
+    {
+        void TestManageRights();
+    }
+
+    public class DefaultNamespaceManager: INamespaceManager
+    {
+        NamespaceManager _manager;
+        public DefaultNamespaceManager(NamespaceManager manager)
+        {
+            _manager = manager;
+        }
+
+        public void TestManageRights()
+        {
+            _manager.GetQueues();
```

Comment:
So I take it there is no Service Bus API to actually query what level rights you have? Bummer...